com.unity.ml-agents/Runtime/Policy/BehaviorParameters.cs

```diff
@@ -56,6 +56,7 @@ public bool useChildSensors
         public string behaviorName
         {
             get { return m_BehaviorName; }
+            set { m_BehaviorName = value; }
```

Comment:
Why allowing setting the behavior name ?

Reply:
Will undo.